reviewer_readme.md

```diff
@@ -1,46 +1,117 @@
+# Introduction
+
 ## Reviewing the Intel Android Hub
-So you've been chosen to review article submissions from Github for this funky fresh blog. I'll guide you through the things you need to do, in order to accept the article with confidence.
+So you've been chosen to review article submissions from Github for this funky fresh blog. This guide will help you through the things you need to do, in order to accept the article with confidence.
+
+--------
+
+## Glossary
+
+- *Pull Requst (PR)* content submissions from other Github users, most commonly Android Hub Contributors.
+- *npm* is a javascript based package manager that installed the Android Hub's dependencies and libraries.
+- *bower* is another javascript based package manager focused on client side (browser) based code.
+- *Continuous Integration (CI)* is an automated process we've setup that looks at a github code update and runs pre-defined checks and tests. The results of the tests are linked into the Pull Request.
+- *Semantic Versioning* is an industry standard versioning system that separates "breaking" changes, from "compatible" changes, from "bugfix" changes. This is typically express as 3 numbers separated by a ".". For example, "3.1.4" is the 3rd major version, 1st minor update to the 3rd version, and the 4th patch version to the 1st minor update.
```

Comment:
I'd provide a link to semantic versioning.  Also -- sentence 2 "This is typically express as" should be "This is expressed as".  In addition to the "ed" being necessary I'd delete typically.  Typically suggests it is up for debate which, it is not if it's governed by semantic versioning.